distdens/fillout.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def fillout(x, y, limits=None, **kwargs):
+    if limits is None:
+        limits = [min(x), max(x), min(y), max(y)]
+    x_inicial = np.array(limits[0]).reshape(1, 1)
+    x_final = np.array(limits[1]).reshape(1, 1)
+    y_inicial = np.array(limits[2]).reshape(1, 1)
+    y_final = np.array(limits[3]).reshape(1, 1)
+    indice_pegado_costa = np.where(x == min(x))[0][0]
+    x_cerrada = np.concatenate(
```

Comment:
Por favor refactoriza en una función las líneas 13-19, pues es código repetido:
```suggestion
    x_cerrada = close_coordinate(x, indice_pegado_costa)
```

Reply:
Muchas gracias por hacer la prueba para refactorizar, @nepito. Eso es más de la mitad del trabajo.